MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -273,10 +276,10 @@
               <Setter Property="BorderBrush" Value="{Binding Path=(wpf:TextFieldAssist.UnderlineBrush), RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" />
             </MultiDataTrigger>
             <Trigger Property="IsPressed" Value="True">
-              <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource PrimaryHueMidBrush}" />
+              <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource MaterialDesign.Brush.Primary}" />
             </Trigger>
             <Trigger Property="IsEnabled" Value="False">
-              <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource MaterialDesignCheckBoxDisabled}" />
+              <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource MaterialDesign.Brush.CheckBox.Disabled}" />
```

Comment:
Isn't it still a bit odd naming-wise. I mean if a consumer decides to change the MaterialDesign.Brush.CheckBox.Disabled brush because he/she wants to affect ONLY the CheckBox control, this now also affects the arrow in the ComboBox. I think the original intention was to provide consumers with more fine-grained control of the brushes; this seems like an odd side-effect IMO.

And I can see there is already such a disabled brush for `ComboBox`, it is used in line 297.

```suggestion
              <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource MaterialDesign.Brush.ComboBox.Disabled}" />
```

Reply:
Fixed